config/sys.config

```diff
@@ -1 +1,37 @@
-[].
+[
+    {lager, [
+        {suppress_supervisor_start_stop, true},
+        {log_root, "/var/log/hpr"},
+        {crash_log, "crash.log"},
+        {handlers, [
+            {lager_file_backend, [
+                {file, "info.log"},
+                {level, info},
+                {size, 0},
+                {date, "$D0"},
+                {count, 7},
+                {formatter_config, [
+                    "time=",
+                    date,
+                    "T",
+                    time,
```

Comment:
We can replace a few of these with a the atom `metadata` from lager.
ref: https://github.com/erlang-lager/lager/blob/master/src/lager_default_formatter.erl#L96-L100